.github/workflows/deployment.yml

```
@@ -0,0 +1,59 @@
+# Copied from https://github.com/notlmn/browser-extension-template/blob/master/.github/workflows/deployment.yml
+name: Deployment
+
+on:
+  workflow_dispatch:
+  # You can manually trigger a deployment on GitHub.com
+  # https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/
+  schedule:
+    - cron: '42 4 * * 4' # At 04:42 every Thursday
+
+jobs:
+  Build:
+    outputs:
+      shouldDeploy: ${{ steps.daily-version.outputs.created }} || github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 20
+      - run: npm ci
+      - uses: fregante/daily-version-action@v1
+        name: Create tag if necessary
+        id: daily-version
+      - uses: notlmn/release-with-changelog@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          template: '{commits}'
+          exclude: '^Meta|^Document|^Readme|^Lint'
+      - name: Update extension’s meta
+        env:
+          VER: ${{ steps.daily-version.outputs.version }}
+        run: npm run version
```

If you remove `$VER` from the `version` script, you can change this to:
```suggestion
        run: npm run version -- ${{ steps.daily-version.outputs.version }}
```

I think it's worth the change. The reason why I used `$VER` initially was because it was shared with the URL.txt file, I think